extensions/amp-analytics/0.1/vendors.js

```diff
@@ -438,6 +438,35 @@ export const ANALYTICS_CONFIG = {
         '${actionParameter}${parameterSuffix}',
     },
   },
+
+  'shinystat': {
+    'requests': {
+      'ssurl': 'https://amp.shinystat.com/cgi-bin/getcod.cgi',
+      'ssparam': 'AMP=1' +
+                 '&USER=${account}' +
+                 '&AMPPAG=${page}' +
+                 '&AMPHR=${canonicalUrl}' +
+                 '&AMPREFER=${documentReferrer}' +
+                 '&AMPRES=${screenWidth}X${screenHeight}' +
+                 '&AMPCOLOR=${screenColorDepth}' +
+                 '&AMPCID=${clientId(AMP_CID)}' +
+                 '&AMPPID=${pageViewId}' +
+                 '&AMPTTL=${title}' +
+                 '&AMPRQC=${requestCount}' +
+                 '&AMPVIE=${viewer}' +
+                 '&AMPSSIDC=${queryParam(ssidc)}' +
+                 '&${ampp1}=${ampv1}' +
+                 '&${ampp2}=${ampv2}',
+      'pageview': '${ssurl}?${ssparam}',
+    },
+    'triggers': {
+      'pageview': {
+        'on': 'visible',
+        'request': 'pageview',
+        'iframePing': true,
```

Comment:
Locally, the easiest way to fake different CDNs is to add an extra mapping in the `/etc/hosts`:
```
127.0.0.1       localhost
127.0.0.1       other.localhost
```
Then you can access the test documents from one hostname and pull in iframes from the other.